_posts/2022-05-30-openmic-workflow-announcement

```diff
@@ -0,0 +1,10 @@
+---
+title: "Upcoming Open Mic - workflow and tooling for front end applications"
+categories: [Open Mic Announcement, Open Mic]
+---
+
+The Open mic session starts on Friday 10-06-20202 at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart her presentation with the topic workflow and tooling for front end applications.
+
+###### Abstract:
+
+The presentation will be about Tooling and workflow for JavaScript libraries. The motivation is that when building a library (package) for another application also still under development, testing, integration and implementation can be rather tedious. Therefore, the aim is to provide some recommendations and guidelines to improve the CI and the workflow between development teams.
```

Comment:
some words are probably missing, does not make much sense. Suggestion:

The presentation is about Tooling and workflow for JavaScript libraries. Building libraries or packages used in another applications still under development may be tedious. Therefore the presentations aims form providing recommendations and guidelines  to improve CI and the workflow between development teams.